### PR TITLE
Applying fixes for the public status-checker.

### PR DESCRIPTION
**Pull request by nibivi77**
### Checklist

- [x] I have tested the changes locally
- [x] I have updated the documentation if necessary
- [ ] I have added/updated tests that prove my fix is effective or that my feature works
- [x] I have checked that my code follows the project's coding style by running `npm run format:check`
- [x] I have checked that my code contains no linting errors by running `npm run lint`
- [x] I have checked that my code contains no type errors by running `npm run typecheck`
- [x] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [x] I have checked that all e2e tests pass by running `npm run test:e2e`

### Test Instructions
Navigate to /status and see the changes.

### CODE REVIEW

File: frontend/public/locales/en/status.json [hunk 1]
```diff
@@ -32,10 +33,10 @@
   },
   "privacy-notice-statement": {
     "summary": "Privacy Notice Statement",
-    "collection-of-use": "The collection and use of your personal information is authorized under the Department of Health Act, the Dental Care Measures Act, the Department of Employment and Social Development Act, and other applicable laws.",
+    "collection-of-use": "The collection and use of your personal information is authorized under the <i>Department of Health Act</i>, the <i>Dental Care Measures Act</i>, the <i>Department of Employment and Social Development Act</i>, and other applicable laws.",
```
Comment (nicholas-ly) [inline frontend/public/locales/en/status.json:36]: See https://github.com/DTS-STN/canadian-dental-care-plan/pull/1736#discussion_r1640111206, which if agreed on, can be applied to other i18n values in this PR.

File: frontend/public/locales/en/status.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "status-checker-content": "The Status Checker is a tool that lets you check the status of your Canadian Dental Care Plan application.",
   "online-status-heading": "Who can check their application status online?",
   "online-status-content": "If you applied for the Canadian Dental Care Plan, you can check your status online.",
+  "terms-conditions": "En utilisant le vérificateur de l'état d'une demande, vous acceptez les conditions générales d'utilisation et à l'énoncé de confidentialité.",
```
Comment (nicholas-ly) [inline frontend/public/locales/en/status.json:7]: The French text is in the `/en/status.json` while the English text is in `/fr/status.json`

File: frontend/public/locales/fr/status.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "status-checker-content": "Il s'agit d'un outil qui vous permet de savoir où en est votre demande de participation au Régime canadien de soins dentaires",
   "online-status-heading": "Qui peut vérifier l'état de sa demande en ligne?",
   "online-status-content": "Si vous avez présenté une demande au titre du Régime canadien de soins dentaires, vous pouvez vérifier l'état de votre demande en ligne.",
+  "terms-conditions": "By using the Status Checker you agree to the following Terms and Conditions of Use and Privacy Notice Statement.",
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/status.json:7]: The French text is in the `/en/status.json` while the English text is in `/fr/status.json`

File: frontend/public/locales/en/status.json (hunk 1 above)
Comment (nicholas-ly) [reply frontend/public/locales/en/status.json]: FYI adding `<cite>` directly to these i18n files will render the HTML tag:
image

We have to pass a `cite` component to a `Trans` to fix this. See https://github.com/DTS-STN/canadian-dental-care-plan/blob/3a7e8d67090849dc55e47e2b047ac35f1442e1dd/frontend/app/routes/%24lang/_public/apply/%24id/terms-and-conditions.tsx#L115

(credit to @AdamP-HRSDC-RHDCC)

Comment (nicholas-ly) [reply frontend/public/locales/en/status.json]: (FYI @nibivi77 I see a commit from today but `frontend/app/routes/$lang/_public/status/index.tsx` hasn't been changed to address the above)

Comment (nibivi77) [reply frontend/public/locales/en/status.json]: OOh ! Sorry I just saw this comment

File: frontend/app/routes/$lang/_public/status/index.tsx [hunk 4]
```diff
@@ -143,15 +143,17 @@ export default function StatusChecker() {
   const microsoftServiceAgreement = <InlineLink to={t('status:links.microsoft-service-agreement')} className="external-link" newTabIndicator target="_blank" />;
   const fileacomplaint = <InlineLink to={t('status:links.file-complaint')} className="external-link" newTabIndicator target="_blank" />;
   const canadaTermsConditions = <InlineLink to={t('status:links.canada-terms-conditions')} className="external-link" newTabIndicator target="_blank" />;
+  const cite = <cite />;
```
Comment (gregory-j-baker) [inline frontend/app/routes/$lang/_public/status/index.tsx:146]: A bit of a nitpick, but I don't think this is necessary.. it just adds noise to the source code.

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**